### ISSUE TITLE
Chmod isn't implemented ?

### ISSUE THREAD

**Issue by BenasPaulikas** [sstur/nodeftpd#60]

**Comment by sstur** [comment]
No. This is not implemented. I'll tag this as a feature request. I'm not sure how this would work on Windows.

**Comment by BenasPaulikas** [comment]
Well this can be integrated to Linux only for now. But It's very important command :)
Thanks
